app/src/main/res/values/dimens.xml

```diff
@@ -214,4 +215,5 @@
     <dimen name="connect_instruction_text_header">18sp</dimen>
 
     <dimen name="sensor_schematic_height">250dp</dimen>
+
```

Comment:
You have checked the item in PR checklist stating that no modifications done at the end of resource files. But here there is a change making merge conflicts.